src/plugins/saved_objects/public/saved_object/helpers/apply_es_resp.ts

```diff
@@ -74,3 +75,55 @@ export async function applyESResp(
   }
   return savedObject;
 }
+
+export async function applyESRespUtil(
```

Comment:
I don't think we need this layer of abstraction - Graph will only hit a very specific subset of the stuff in there (e.g. it won't ever contain `kibanaSavedObjectMeta` or an index pattern that has to be hydrated.

The logic with the mapping and `_deserialize` is also just used to call `JSON.parse` on the workspace state - something we should just do inline.

Reply:
Sure, we don't need it.
I updated the PR.